### DISCUSSION TITLE
Correct "Melon" to "Watermelon"

### DISCUSSION THREAD

**Discussion by TheGuizai** [BedrockTweaks/Files#329] [Ideas]
The two fruits are very different. Although being from the same "melon" family, it is not correct to call watermelons melons (unless you call Minecraft cows "bovines" instead of, well, "cow"), and I have seen a LOT of people who do not know the difference!

The Minecraft "melon" is a watermelon - red inside, strong green outside, and black seeds spread in the fruit. Melons have yellow or faint green outside, orange or green fruit, and its seeds are of a similar color but packed in the center of the melon with a fiberous tissue.

Sure, a retexturing could be done, but the appearance of melons is great and distinct, so I propose a rename.

**Comment by BuxBux1** [comment]
this is so cool frfr

**Comment by SquatchHunter** [comment]
#93 
I thought I saw this being suggested before...
I also thought I had replied, whoops.

You dont need to remake suggestions because they get old, we see them.
